### PR TITLE
Add test id for MoreOptions button

### PR DESCRIPTION
**Pull request by mariana-mendes**
#### What is the purpose of this pull request?
as the title says.

#### What problem is this solving?
Avoid use css selectors in test implementations.

[Workspace](https://mariana--cosmetics1.myvtex.com/admin/collections/1006/).
#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/FilterBar/index.js [hunk 1]
```diff
@@ -217,6 +219,7 @@ export const filterBarPropTypes = {
   submitFilterLabel: PropTypes.string,
   /** New Filter title label for inside the 'More options' menu */
   newFilterLabel: PropTypes.string,
+  testIdMoreOptions: PropTypes.string,
```
Comment (matheusps) [inline react/components/FilterBar/index.js]: Why do we need to pass it by props every time? 🤔. I think the internal components should have static ids.

Comment (mariana-mendes) [reply react/components/FilterBar/index.js]: Because we can use the component more than once in a app, then, they should have a unique key, the id.

**Comment by mariana-mendes** [comment]
> Can't we use the other queries?
> 
> Test Id should be the last resort:
> https://testing-library.com/docs/guide-which-query

The intention is use native queries from cypress. :thinking:

**Comment by emersonlaurentino** [comment]
we already had a discussion about this in #848 and we did some PRs in #856 

in short: it would be to use the `testId` prop and the attribute in the element to be `data-testid` because it is an attribute that can be read by Cypress and the Testing Library.

File: react/components/FilterBar/index.js (hunk 1 above)
Comment (matheusps) [reply react/components/FilterBar/index.js]: Got it, makes sense.

File: react/components/FilterBar/index.js [hunk 2]
```diff
@@ -217,6 +220,10 @@ export const filterBarPropTypes = {
   submitFilterLabel: PropTypes.string,
   /** New Filter title label for inside the 'More options' menu */
   newFilterLabel: PropTypes.string,
+  testIds: PropTypes.shape({
+    moreOptionsButton: PropTypes.string,
+    submitFiltersButton: PropTypes.string,
+  }),
```
Comment (rerissondaniel) [inline react/components/FilterBar/index.js]: These should be inside a `moreOptionsFilterTag` object, no? This way if we wanted to add ids to clearAllFilterButton we could only add a new prop to testIds

**Pull request merged**